apps/financial-aid/backend/src/app/lib/userContextExtractor.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  decodeToken,
+  IDENTITY_SERVER_SESSION_TOKEN_COOKIE_NAME,
+  User,
+} from '@island.is/financial-aid/shared/lib'
+import { ExecutionContext } from '@nestjs/common'
+import { GqlExecutionContext } from '@nestjs/graphql'
+
+export const getUserFromContext = (
+  context: ExecutionContext & { contextType?: string },
+): User | undefined => {
```

Comment:
This is almost the same code as in `current.user.decorator`. Should this be in a lib?

Reply:
Well you are not wrong 😅  I simplified this some, let me know if you still feel strongly about it. Currently we only have `financial-aid-shared`, not sure if this should be there.